### ISSUE TITLE
Search Filter does not clear when item is selected

### ISSUE THREAD

**Issue by cgeo7** [angular-ui/ui-select#1083]
Even though reset-search-input="true" it does not work in the latest version

**Comment by MiguelMike** [comment]
+1

**Comment by MiguelMike** [comment]
I'd really love to have this working. For now you can use the following as a workaround:

In template add the following (change `ctrl.` to match your situation) to your `ui-select` directive:

```
on-select="ctrl.resetInput($select)"
```

Next, in your controller add:

```
ctrl.resetInput = function($search) {
	$search.search = '';
}
```

Refer to docs for more information on `on-select` attribute.

**Comment by Sheeryada** [comment]
+1 for the issue. 
@MiguelMike thanks for the workaround.

**Comment by pdoreau** [comment]
@MiguelMike it works with a flickering effect as selection is added just before the text is removed
+1 for an internal and clean fix !

**Comment by wesleycho** [comment]
Can you post a Plunker of the problem?

**Comment by user378230** [comment]
Closing due to lack of plunkr and possibly resolved by #1746 anyway.
